components/post_view/post_body/post_body.jsx

```diff
@@ -173,7 +173,7 @@ export default class PostBody extends React.PureComponent {
             </React.Fragment>
         );
 
-        const hasPlugin = post.type && this.props.pluginPostTypes.hasOwnProperty(post.type);
+        const hasPlugin = (post.type && this.props.pluginPostTypes.hasOwnProperty(post.type)) || (post.props && post.props.type);
```

Comment:
Should the `post.props.type` be checked with `pluginPostTypes.hasOwnProperty`?

Reply:
I think so.  Good idea.